### DISCUSSION TITLE
Error building @react-navigation/native

### DISCUSSION THREAD

**Discussion by Cupidazul** [necolas/react-native-web#2366] [Help]
I was trying to use @react-navigation/native 6.0.11 and I got the following error when building "react-scripts build":

Attempted import error: 'default'.'interpolate' is not exported from 'react-native-reanimated' (imported as 'Animated').

using: "react-native-web": "^0.18.7"  +   "@react-navigation/drawer@6.4.3" + "react-native-reanimated": "^2.9.1"

related:
https://github.com/software-mansion/react-native-reanimated/discussions/3459

**Comment by ayozebarrera** [comment]
I had same issue because i was using interpolate from reanimated. Updating some packages, I needed to use the new interpolateNode from reanimated

```jsx
  //old way
  const scale = Animated.interpolate(drawerProgress, {
    inputRange: [0, 1],
    outputRange: [1, 0.8],
  });
 
  //new way
  const scale = Animated.interpolateNode(drawerProgress, {
    inputRange: [0, 1],
    outputRange: [1, 0.8],
  });
```

Hope this helps you
